collect_app/src/main/AndroidManifest.xml

```diff
@@ -286,7 +286,7 @@ the specific language governing permissions and limitations under the License.
 
         <meta-data
             android:name="com.google.android.geo.API_KEY"
-            android:value="AIzaSyBS-JQ-dnaZ_8qsbvSyr_I3rTPFd5fJsYI" />
+            android:value="AIzaSyD1D8OvyArqGg0OezuEPrx8U08uYb4e7XY" />
```

Comment:
Please revert the changes to this file. You do need your own key locally but it can't be checked in.